doc/releasenotes/16_0_0_summary.md

```diff
@@ -246,4 +246,13 @@ VSchema Example
 
 #### Flag Deprecations
 
-The flag `lock-shard-timeout` has been deprecated. Please use the newly introduced `lock-timeout` instead. More detail [here](#lock-timeout-introduction).
\ No newline at end of file
+The flag `lock-shard-timeout` has been deprecated. Please use the newly introduced `lock-timeout` instead. More detail [here](#lock-timeout-introduction).
+
+### VTTestServer
+
+#### Improvement
+
+Creating a database takes a lot longer in vttestserver. This can be problematic in the test environment where tests-cases do a lot of `create` and `drop` database.
+In an effort to minimize the database creation time, we have changed the value of `tablet_refresh_interval` to 10s while instantiating vtcombo during vttestserver initialization.
+For any production cluster the default value of this flag is still [1 minute](https://vitess.io/docs/15.0/reference/programs/vtgate/). Reducing this values might put more stress on Topo Server (since we now read from topo server more often) but for testing purpose 
```

Comment:
```suggestion
For any production cluster the default value of this flag is still [1 minute](https://vitess.io/docs/15.0/reference/programs/vtgate/). Reducing this values might put more stress on Topo Server (since we now read from topo server more often) but for testing purposes 
```